src/Swashbuckle.AspNetCore.Swagger/Swashbuckle.AspNetCore.Swagger.csproj

```
@@ -19,7 +19,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.OpenApi" Version="1.1.4" />
+    <PackageReference Include="Microsoft.OpenApi" Version="[1.1.4,1.2.0)" />
```

```suggestion
    <PackageReference Include="Microsoft.OpenApi" Version="1.2.2" />
```

We fixed it at the source, Microsoft released a version without the breaking changes. Swashbuckle can now target 1.2.2